tests/v2/validation/workloads/workload.go

```diff
@@ -0,0 +1,18 @@
+package workloads
+
+import (
+	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/extensions/workloads"
+	deployment "github.com/rancher/shepherd/extensions/workloads/deployment"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	defaultNamespace = "default"
+)
+
+func CreateDeployment(steveclient *steveV1.Client, containersTemplate []corev1.Container, containerName string) (*steveV1.SteveAPIObject, error) {
```

Comment:
should this be a private function? `createDeployments` as this is only a helper? I'm thinking if its essential to have this function? Instead we can have this added in the test itself right why have a function ?

Reply:
I can add it in the test